modules/azure/disk.go

```diff
@@ -0,0 +1,100 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-07-01/compute"
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// DiskExists indicates whether the specified Azure Managed Disk exists
+// This function would fail the test if there is an error.
+func DiskExists(t testing.TestingT, diskName string, resGroupName string, subscriptionID string) bool {
+	exists, err := DiskExistsE(diskName, resGroupName, subscriptionID)
+	require.NoError(t, err)
+	return exists
+}
+
+// DiskExistsE indicates whether the specified Azure Managed Disk exists
+func DiskExistsE(diskName string, resGroupName string, subscriptionID string) (bool, error) {
+	// Get the Disk object
+	_, err := GetDiskE(diskName, resGroupName, subscriptionID)
```

Comment:
These two methods don't seem useful.  I would get the same result if I called getdisk with an invalid diskname.

Reply:
There is an update to this check, see networkinterface.go in Master.

Reply:
That makes sense, we need to follow same pattern on all exists methods.  Otherwise they are redundant.